tests/sysauth/sssd.pm

```diff
@@ -15,6 +15,7 @@ sub run() {
     /;
     script_run "systemctl stop packagekit.service; systemctl mask packagekit.service";
     script_run "zypper -n refresh && zypper -n in @test_subjects";
+    wait_idle;
```

Comment:
or better use assert_script_run() which actually waits until the command completed successfully.